helpers/build_packages.sh

```diff
@@ -83,6 +83,9 @@ while true; do
               *) BUILDPKG_PATH=$2;;
           esac
           shift;;
+      --build-all)
+          BUILDALL=1
```

Comment:
When `--build-all` is not specified in command line, the script still builds all packages.

Also please mention this new flag in the `usage()`.

Reply:
Thanks for rebase, I took those changes and ran the script without --build-all again, and it still builds all packages:)